schema/show.js

```diff
@@ -315,7 +315,7 @@ const Show = {
       .then(show => {
         if (!show.displayable && !show.is_reference) return new Error('Show Not Found');
         return show;
-      });
+      }).catch(() => null);
```

Comment:
Nope those errors get propagated!

This just basically swallows the 404s you'd get from `/api/v1/show/invalid-show`, reason for that being we have an endpoint that might return already deleted documents, that fail on further fetching.